### DISCUSSION TITLE
EP Check for Updates not Working

### DISCUSSION THREAD

**Discussion by tallman69** [valinet/ExplorerPatcher#1981] [Q&A] [answered]
On one of my computers I cannot check for updates for EP from the EP properties menu.  I have tried uninstalling and reinstalling EP and deleting the two registry keys: `Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher` and
`Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher` to no avail.  I am not having this problem on my two other computers with the same Windows 11 and EP builds. Otherwise EP is working just fine.

OS Build:  22621.2134
EP Build:  22621.1992.56.3

**Comment by tallman69** [comment, marked as answer]
I figured out that the problem was not with EP at all. I had notifications for File Explorer toggled off In the Notifications and Actions settings, so I was not getting a notification when I checked for updates from the EP Properties menu.

**Comment by pyrates999** [comment]
Please mark your response as the answer then.

**Reply by tallman69** [reply]
I do not see the `Mark as answer` button at the bottom of my reply.  Is there another way to accomplish this?

**Reply by Poopooracoocoo** [reply]
It could be because this discussion is not in the Q&A section

**Reply by pyrates999** [reply]
you're right!

**Reply by tallman69** [reply]
Thanks!  Moved this to Q&A and marked it as answered.
